overview-generator/template_index.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+ <title>mundialis GRASS GIS Addons Manual pages</title>
+ <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+ <meta name="Author" content="GRASS Development Team">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+ <link rel="stylesheet" href="grassdocs.css" type="text/css">
+</head>
+<body bgcolor="#FFFFFF">
+<h2>mundialis GRASS GIS Addons Manual pages</h2>
+
+<table><tr><td>
+<img src="https://www.fair-opendata.de/wp-content/uploads/2020/03/mundialis.png" width=200>
+</td><td>
+<a href="https://grass.osgeo.org">GRASS GIS</a> is free software,
+anyone may develop his/her own extensions (addons). The addons listed
+here are currently not part of the core software package.<br>
+Addons can easily be <b>installed</b> in your local GRASS GIS installation
+through the graphical user interface (<i>Menu - Settings - Addons
+Extension - Install</i>) or via the <a
+href="https://grass.osgeo.org/grass-stable/manuals/g.extension.html">g.extension</a> command. <p> <i>These
+manual pages are updated daily. Last run: {{ date }}</i>
+<p> How to contribute?
+<p> See instructions here:
+<a href="https://apps.mundialis.de/grass-addons/How-to-create-a-GRASS-GIS-addon.md">How to create a GRASS GIS addon</a>.
```

Comment:
Also available more nicely rendered here: https://github.com/mundialis/grass-gis-helpers/blob/main/How-to-create-a-GRASS-GIS-addon.md